### PR TITLE
Show some poetry in the toplevel

### PR DESCRIPTION
**Pull request by haselwarter**

### CODE REVIEW

File: src/utils/haiku.ml [hunk 1]
```diff
@@ -0,0 +1,162 @@
+(* courtesy of http://www.cl.cam.ac.uk/research/hvg/haiku.html *)
+let haikus = [
```
Comment (andrejbauer) [inline src/utils/haiku.ml]: Glad to see you've got your priorities straight.

**Comment by haselwarter** [comment]
Don't worry, I'm also working on a mail client.

**Comment by andrejbauer** [comment]
How set are you on getting this in? I'd rather Andromeda with "humor" lest the aliens be offended.

**Comment by andrejbauer** [comment]
The poetry is not of high enough value, but I would accept a video of an interpretative dance about type theory.

**Comment by jonsterling** [comment]
:+1: for interpretive type theory dancing

**Pull request closed without merging**